source/Expression/ExpressionSourceCode.cpp

```diff
@@ -236,8 +239,16 @@ bool ExpressionSourceCode::SaveExpressionTextToTempFile(
     break;
   }
 
+#if defined(_WIN32)
```

Comment:
If making temp files needs abstraction for other systems we should look and see if there is a LLVM equivalent. We don't want every place in the LLVM/Clang/LLDB source base to do any "#if defined(_WIN32)"